SPECS/coreutils/coreutils.spec

```diff
@@ -1,21 +1,25 @@
 Summary:        Basic system utilities
 Name:           coreutils
 Version:        8.30
-Release:        6%{?dist}
+Release:        7%{?dist}
 License:        GPLv3
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Base
 URL:            https://www.gnu.org/software/coreutils
-Source0:        http://ftp.gnu.org/gnu/coreutils/%{name}-%{version}.tar.xz
+Source0:        https://ftp.gnu.org/gnu/%{name}/%{name}-%{version}.tar.xz
 # make this package to own serial console profile since it utilizes stty tool
 Source1:        serial-console.sh
-Patch0:         http://www.linuxfromscratch.org/patches/downloads/coreutils/coreutils-8.30-i18n-1.patch
-Patch1:         http://www.linuxfromscratch.org/patches/downloads/coreutils/coreutils-8.10-uname-1.patch
+Patch0:         coreutils-8.30-i18n-1.patch
```

Comment:
We should list the patch source in a comment at least.

Reply:
Added the provenance of these patches in the latest commit